website/blog/2020-06-27-lithium.md

```diff
@@ -0,0 +1,503 @@
+  based on the filesystem information
+
+```scala
+//main.sc
+import $file.hello-world
+val greeting = hello-world.greeting
+
+//hello-world.sc
+val greeting = "Hello world!"
+```
+
+- importing dependecies
+
+```scala
+import $ivy.`com.lihaoyi::scalatags:0.7.0`
+import scalatags.Text.all._
+
+val rendered = div("Moo").render
+```
+
+## Worksheet dependency support
+
+In addition to Ammonite support, which will be useful to anyone interested in
+Scala scripting, we've also improved the support for Metals Scala worksheets.
+Thanks to the efforts of [olafurpg](https://github.com/olafurpg), we now are
+able to provide importing dependencies inside your Scala worksheets with
+completions and diagnostics working right from the bat.
+
+![worksheet-imports](https://i.imgur.com/qXgdNWM.gif)
+
+You can add them using the following syntax:
+
+```
+$dep.`organisation::artifact:version`
+```
+
+for example:
+
+```
+$dep.`com.lihaoyi::scalatags:0.9.0`
+```
+
+`::` is the same as `%%` in sbt, which will append the current scala binary
+version to the artifact name.
+
+## New project provider
+
+Creating new Scala projects is usually a bit of work to get started, which is a
+problem for both beginners and experts alike. Normally you don't want a lot of
+time to figure out the best setup for starting with a new technology or the
+language itself.
+
+There are great tools like [giter8](http://www.foundweekends.org/giter8/), which
+is used in the `sbt new` command, that enables people to use already existing
+Scala templates found on Github such as https://github.com/scala/scala-seed.g8.
+The problem is when you don't use sbt or you don't know the exact template you
+want to use.
+
+In this version we added an additional command called `New Scala Project` that
+enables users to reuse the same mechanism in comfort of your own UI. Firstly,
+you can choose from existing templates, either curated by the Metals team, ones
+existing on the giter8
+[wiki page](https://github.com/foundweekends/giter8/wiki/giter8-templates) or
+alternatively an entirely custom templates. Then you can choose the exact
+location for the new project and the name. Afterwards, Metals will run giter8
+itself and ofer to open the new project.
+
+![new-project](https://user-images.githubusercontent.com/3807253/81842954-2d1b3680-954d-11ea-934a-96fe53cafeb9.gif)
+
+The command is also available as a button in the welcome view of the Visual
+Studio Code Metals extension, which means it will be visible in case there are
+no imported projects in the current workspace.
+
+The first curated templates were choosen by the team, however we are happy to
+include more so that it's easier for people to discover new amazing libraries or
+frameworks.
+
+## Miscellaneous improvements
+
+- fix run/debug in Maven workspaces
+- always run newest sbt version if none is specified
+- fix find references for Scala 3 on Windows
+- pick jvm targets before other when running
```

Comment:
I'm actually a bit confused at what this means.

Reply:
Och yeah, that wasn't too clear :sweat_smile: